draft-ietf-mmusic-dtls-sdp.xml

```diff
@@ -355,6 +358,21 @@
                 association. The combination of the SDP 'tls-id' attribute values of the SDP
                 offerer and answerer identifies each individual DTLS association.
             </t>
+            <t>
+                NOTE: There are cases where the SDP ‘tls-id’ attribute value generated by the
+                offerer will end up being used for multiple DTLS associations. For that reason
+                the combination of the attribute values of the offerer and answerer is needed
+                in order to identity a DTLS association. An example of such case is where the
+                offerer sends an update offer (<xref target="sec-oa-mod"/>), without modifying its
+                attribute value, but the answerer determines that a new DTLS association is to
+                be created. The answerer will generate a new local attribute value for the new
+                DTLS association (<xref target="sec-oa-answer"/>), while the offerer will use the
+                same attribute value that it used for the current association. Another example is
+                when the Session Initiation Protocol (SIP) <xref format="default" pageno="false"
+                target="RFC3261"/> is used for signalling, and an offer is forked to multiple answerers.
+                The attribute value generated by the offerer will be used for DTLS associations
+                established by each answerer.
+            </t>

```

Comment:
The 1-800-GoFedEx example would still work after the answer in 183 Session Progress response.  SIP early media does not have to be equal to media before the SDP answer. I understand what you are trying to accomplish, but I do not think that NAT traversal, security and consent to send are compatible with media before answer.  So far, the best solution for the early media problem that I saw was based around setting a media connection to SBC before the call, sending early media (with potential transcoding and re-encryption) through the SBC, and then taking SBC out of media path once connection is established using 3pcc. This solution requires a bit more resources but it does not require unauthenticated media or handshakes with unverified party. The only risk is early media intercept and recording by SBC.